views/pages/index.ejs

```diff
@@ -2,6 +2,6 @@
 <% include ../partials/header %>
 </head>
 <div class="centered">
-	<h1 id="header">Technoramic 5190</h1>
-	<em>We are the co-ed high school team of MICDS, non-profit and independent school in Saint Louis, Missouri.</em>
+	<h1 id="header">Welcome to our website!</h1>
+	<em>Technoramic, or Team #5190, is the high school co-ed FTC and Relic Recovery Team of MICDS. Click the links above to see what we're about, see our current construction or programming progress, or contact us. </br>-Technoramic</em>
```

Comment:
where does this </br> start? if there isn't a start, it might not get rendered. Also, -Technoramic is kind of implied here, and not really necessary